### DISCUSSION TITLE
Is there a way of launching SDDM when switching to desktop mode?

### DISCUSSION THREAD

**Discussion by ChocolateLoverRaj** [Jovian-Experiments/Jovian-NixOS#399] [Q&A]
Can I get to the SDDM screen where I can select a user and login as other users instead of directly going into KDE? For security reasons, I have a user called "gamer" which is used for Steam big picture mode with Jovian. For desktop mode, it's more useful if I can login as my user ("rajas").

**Comment by K900** [comment]
No.
